package/kernel/mac80211/files/lib/wifi/mac80211.sh

```diff
@@ -106,6 +108,8 @@ detect_mac80211() {
 			dev_id="set wireless.radio${devidx}.macaddr=$(cat /sys/class/ieee80211/${dev}/macaddress)"
 		fi
 
+		ssid="OpenWrt_11${mode_band}_$(macaddr_geteui $(cat /sys/class/ieee80211/${dev}/macaddress))"
```

Comment:
This would become:
```
local labelmac=$(get_mac_label)
local ssid=OpenWrt_11${mode_band}
[ -n "$labelmac" ] && ssid="${ssid}_$(macaddr_geteui $labelmac)"